app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav class="">
+  <i class="fa-solid fa-chevron-left"><%= link_to 'Back', categories_path, class: 'your-link-class' %></i>
+</nav>
+<h1>New expense</h1>
+
+<%= render "form", expense: @expense %>
+
+<br>
```

Comment:
- [ ] After clicking on the Submit button or Back button on the "Add a new transaction" page, It should redirect the users to the transactions page(expenses page) for that category as requested in the project requirements but it's redirecting users to the home page(category page). Please make sure to fix this issue.

![image](https://github.com/Othniel2471/budget_app/assets/83803414/7a123087-bde1-4483-bf3b-2f747a45b223)